contracts/USM.sol

```diff
@@ -131,4 +148,75 @@ contract USM is BufferedToken {
         latestFumPrice = fumPrice();
         emit LatestFumPriceChanged(previous, latestFumPrice);
     }
+
+    /** INTERNAL FUNCTIONS */
+
+    /**
+     * @notice Funds the pool with ETH, minting FUM at its current price
+     */
+    function _fund(address to, uint ethIn) internal {
+        uint _fumPrice = fumPrice();
+        uint fumOut = ethIn.wadDiv(_fumPrice);
+        ethPool = ethPool.add(ethIn);
```

Comment:
Have a go at removing `ethPool` and using `address(this).balance` instead.

Reply:
Yeah seems worth a shot...  This is the kind of refactoring our tests are for right!